hera_cal/utils.py

```diff
@@ -181,7 +181,7 @@ def JD2LST(JD, longitude=21.42830):
     LST = []
     for jd in JD:
         # construct astropy Time object
-        t = Time(jd, format='jd', scale='utc')
+        t = Time(jd, format='jd')
```

Comment:
What is the motivation for removing `scale='utc'`? Most examples I've seen online specify this, but admittedly I'm not familiar enough with the other options to know if this is correct or not.

Reply:
So 1) the default is `utc` I believe, however that doesn't mean we shouldn't have it there, but 2) I was just going to follow the `pyuvdata` convention which calls it without a `scale` parameter. We can leave it in if you want. I'm happy either way